dds_registration/models.py

```diff
@@ -176,6 +176,8 @@ class Payment(Model):
     ]
     DEFAULT_METHOD = "INVOICE"
 
+    method = models.TextField(choices=METHODS, default=DEFAULT_METHOD)
```

Comment:
Please revert this, we already store this in `data`:

- https://github.com/Depart-de-Sentier/dds_registration/blob/main/dds_registration/views/membership.py#L44
- https://github.com/Depart-de-Sentier/dds_registration/blob/main/dds_registration/views/event_registration.py#L86

Reply:
I don't agree with this (hiding data in hard-to-access entities isn't a good coding approach), but ok: going to create helpers to extract them from there.